include/seqan3/alphabet/structure/RnaStructureAlphabet.hpp

```diff
@@ -34,7 +34,7 @@
 
```

Comment:
don't rename this file to RnaStructureAlphabet.hpp, files should always stay snake_case. Maybe just rename it to concept.hpp like in the other alphabet folders?